### ISSUE TITLE
Improve version checking on lcustom linting

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#4796]
We're hard-coding the version here:

https://github.com/tridactyl/tridactyl/blob/80d00d4db7bcc8f07487b386ea79cebf76ff5b12/custom-eslint-rules/unsupported-apis.js#L4

But we should read it from here, at least for Firefox:
https://github.com/tridactyl/tridactyl/blob/80d00d4db7bcc8f07487b386ea79cebf76ff5b12/src/manifest.json#L143C7-L143C7

**Comment by bovine3dom** [comment]
@glacambre https://github.com/tridactyl/tridactyl/blob/80d00d4db7bcc8f07487b386ea79cebf76ff5b12/custom-eslint-rules/unsupported-apis.js#L35 means we're using the same version for all the supported browsers, right?

**Comment by glacambre** [comment]
@bovine3dom Indeed! It shouldn't be too hard to adapt the code though, I believe replacing `minimalSupportedFirefoxVersion` with a dictionnary whose keys would be `chrome` or `firefox` and whose values would be the corresponding versions and then indexing into that using `browser` should work.

114 is the most recent firefox ESR btw :)

**Comment by bovine3dom** [comment]
That's exactly what I thought, thanks!

**Comment by bovine3dom** [comment]
Added in #5432
